browser/ui/views/sidebar/sidebar_items_contents_view.cc

```diff
@@ -469,8 +473,11 @@ void SidebarItemsContentsView::UpdateItemViewStateAt(size_t index,
                sidebar::SidebarItem::BuiltInItemType::kChatUI;
       };
 
-      if (is_ai_chat(item) && item_view->GetEnabled()) {
-        item_view->SetEnabled(false);
+      if (is_ai_chat(item)) {
+        if (item_view->GetEnabled()) {
+          item_view->SetEnabled(false);
+        } else {
+        }
```

Comment:
I believe this change is not needed. Could you revert this?

Reply:
ah yes, my bad, it was a left over from an experiment I was doing. will remove